src/components/Composer.vue

```diff
@@ -161,9 +161,9 @@
 							t('mail', 'Add attachment from Files')
 						}}
 					</ActionButton>
-					<ActionButton :disabled="encrypt" icon="icon-folder" @click="onAddCloudAttachmentLink">
+					<ActionButton :disabled="encrypt" icon="icon-public" @click="onAddCloudAttachmentLink">
 						{{
-							t('mail', 'Add attachment link from Files')
+							t('mail', 'Add share link from Nextcloud Files')
```

Comment:
this doesn't work with theming. could you add a placeholder in the translation string and substitude with ``OC?.theme?.name ?? 'Nextcloud'``? Then it should pick up the theming name and fall back to Nextcloud if nothing is set.

Reply:
doesnt everyone calls their cloud Nextcloud? :P

Reply:
definitely not ;)